src/Core/Suppressions/IFilterableIssue.cs

```diff
@@ -18,9 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Collections.Generic;
-
-namespace SonarLint.VisualStudio.Core
+namespace SonarLint.VisualStudio.Core.Suppression
```

Comment:
The suppression-related interfaces are now all in the namespace, and in a folder called "Suppression".